src/ol_infrastructure/components/services/k8s.py

```diff
@@ -468,19 +474,28 @@ def __build_route_list(
 
 class OLApisixOIDCConfig(BaseModel):
     application_name: str
-    k8s_namespace: str
     k8s_labels: dict[str, str] = {}
+    k8s_namespace: str
     vault_mount: str = "secret-operations"
-    vault_path: str
     vault_mount_type: str = "kv-v1"
+    vault_path: str
     vaultauth: str
 
-    oidc_scope: str = "openid profile email"
     oidc_bearer_only: bool = False
     oidc_introspection_endpoint_auth_method: str = "client_secret_basic"
-    oidc_ssl_verify: bool = False
     oidc_logout_path: str = "/logout/oidc"
     oidc_post_logout_redirect_uri: str = "/"
+    oidc_refresh_session_interval: NonNegativeInt = 1800
+    oidc_renew_access_token_on_expiry: bool = True
+    oidc_scope: str = "openid profile email"
```

Comment:
This might be better as a list that gets `" ".join`ed to pass through so that it's easier to modify